scipy/stats/stats.py

```diff
@@ -5421,22 +5421,26 @@ def combine_pvalues(pvalues, method='fisher', weights=None):
     ----------
     pvalues : array_like, 1-D
         Array of p-values assumed to come from independent tests.
-    method : {'fisher', 'stouffer'}, optional
+    method : {'fisher', 'pearson', 'tippett', 'stouffer', 'mudholkar_george'},
+    optional.
         Name of method to use to combine p-values. The following methods are
         available:
 
         - "fisher": Fisher's method (Fisher's combined probability test),
-          the default.
+          the default, the sum of the logarithm of the p-values.
+        - "pearson": Pearson's method (similar to Fisher's but uses sum of the
+          complement of the p-values inside the logarithms).
+        - "tippett": Tippett's method (minimum of p-values).
         - "stouffer": Stouffer's Z-score method.
+        - "mudholkar_george": the difference of Fisher's and Pearson's methods.
+          Uses a t-distribution approximation to the logistic [7]_.
```

Comment:
Let's remove this line here and add a more detailed comment in the notes, maybe: "The test statistic of … is the sum of logisitic random variables and equation 3.6 in … is used to approximate the p-value based on Student's t-distribution"

Reply:
done.